core/src/raw/tests/utils.rs

```diff
@@ -80,7 +80,14 @@ pub fn init_test_service() -> Result<Option<Operator>> {
     // Enable blocking layer if needed.
     if !op.info().full_capability().blocking {
         let _guard = TEST_RUNTIME.enter();
-        op = op.layer(layers::BlockingLayer::create().expect("blocking layer must be created"))
+        #[cfg(feature = "layers-blocking")]
```

Comment:
We don't need those since `raw/tests` is hide under feature `tests` which enables `layers-blocking` already.

Reply:
Yep. It's an intermediate change .. Let me revert it.